src/index.cpp

```diff
@@ -99,11 +100,10 @@ Index<T, TagT, LabelT>::Index(Metric m, const size_t dim, const size_t max_point
         std::memset(_pq_data, 0, total_internal_points * _num_pq_chunks * sizeof(char));
     }
 
-    _start = (uint32_t)_max_points;
```

Comment:
add these back, since _Start should be a member of index class.

Reply:
in save_graph we have 
```
uint32_t ep_u32 = _start;
    out.write((char *)&ep_u32, sizeof(uint32_t));
```

Do you think we can omit writing_start as well ?

Reply:
I believe the medtadata relting to fz_points and are _start are very crucial to search. So we need a way to pass in those values while saving during build. so that we can use them while load.